src/core/components/radio/index.tsx

```diff
@@ -129,6 +138,8 @@ interface RadioProps extends InputHTMLAttributes<HTMLInputElement>, Props {
 	defaultChecked?: boolean
 	label?: ReactNode
 	supporting?: ReactNode
+	error: boolean
```

Comment:
What does it mean to have an error at the radio button level? We're already capturing errors at the radio group level (for instance, to indicate that the user must select an option). Is it useful to indicate that an individual radio button has been selected erroneously?